### PR TITLE
Added @kayhayen (Nuitka's author) to the maintainers

### PR DESCRIPTION
**Pull request by frol**

### CODE REVIEW

**Comment by ocefpaf** [comment]
@frol ideally you should fork and send the PR. Otherwise, as you probably noticed. You'll fire up the CIs on master and the file will be uploaded.  That might not be desirable if you are testing a new feature/version.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -48,3 +48,4 @@ about:
 extra:
   recipe-maintainers:
     - frol
+    - kayhayen
```
Comment (ocefpaf) [inline recipe/meta.yaml]: :+1:

**Comment by frol** [comment]
@ocefpaf Yeah, I have already noticed that. Sorry...

**Comment by ocefpaf** [comment]
No problem. (I've done that several times until I got it right :wink: )

Also feel free to merge immediately changes like that. No need to wait for CIs in this case.

**Pull request merged**